src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/Example8.java

```diff
@@ -0,0 +1,17 @@
+/*
+WhitespaceAround
+allowEmptyLambdas = true
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.whitespacearound;
+
+class Example8 {
+  void example() {
+    // xdoc section -- start
```

Comment:
Why are we removing so much context from these examples, i.e. class/method declaration? With each check having it's own page now, I don't think we are pressed for space. And if we are trying to save space - we do not need method definitions in many of these examples.

Reply:
> Why are we removing so much context from these examples, i.e. class/method declaration?

I said something similar in another PR somewhere in this mess. Some of these contexts make reading the code easier to understand, like am I in a method and I am looking at variable declarations, or am I in a class and am look at field declarations.

Reply:
I don't see us hurting for space that we can't allow 2 more lines after method definition to show class. I thought original conversation was we don't want imports/package (if context is not needed), but wanted everything else.

Reply:
Ok, lets not discuss this too much.

@stoyanK7 , please keep class also under start comment

Reply:
Done.